src/Azure/Foundatio.AzureServiceBus.csproj

```diff
@@ -9,8 +9,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Foundatio.AzureServiceBus</RootNamespace>
     <AssemblyName>Foundatio.AzureServiceBus</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
Thought about that, but we are trying to be as downward compat as possible which is why we were sill on 4.5.0.  The only reason I went to 4.5.1 is because it was required for using the logging nuget package.